debian/postinst

```diff
@@ -13,6 +13,12 @@ case "$1" in
         # Create Symlink
         ln -s /usr/share/sonic-pi/app/gui/qt/sonic-pi /usr/bin/make-music
 
+        if `dpkg --compare-versions $2 lt 2.9-0`; then
```

Comment:
@convolu Creating `test.sh`:

```bash
#!/bin/bash

if `dpkg --compare-versions $1 lt 2.9-0`; then
    echo "Pass"
else
    echo "Fail"
fi
```

and running `./test.sh 2.8` outputs `Pass` but `./test.sh 2.9` outputs `Fail`.